pghoard/pghoard.py

```diff
@@ -287,24 +295,25 @@ def start_walreceiver(self, site, chosen_backup_node, last_flushed_lsn):
     def _get_site_prefix(self, site):
         return self.config["backup_sites"][site]["prefix"]
 
-    def create_backup_site_paths(self, site):
+    def create_backup_site_paths(self, site: str) -> BackupSitePaths:
         site_path = os.path.join(self.config["backup_location"], self._get_site_prefix(site))
         xlog_path = os.path.join(site_path, "xlog")
+        timeline_path = os.path.join(site_path, "timeline")
         basebackup_path = os.path.join(site_path, "basebackup")
 
-        paths_to_create = [
-            site_path,
-            xlog_path,
-            xlog_path + "_incoming",
-            basebackup_path,
-            basebackup_path + "_incoming",
-        ]
+        backup_site_paths = BackupSitePaths(
+            uncompressed_files_path=xlog_path + "_incoming",
```

Comment:
New code does not create `site_path`, probably should.

Reply:
It should :thinking: `os.makedirs` creates subdirectories that do not exist. In theory I just added a directory for `timeline` (which was not created by default)

Reply:
Oops, sorry, my bad, I forgot `os.makedirs` is recursive